erlang/content.md

```diff
@@ -0,0 +1,28 @@
+# What is Erlang?
+
+Erlang is a programming language used to build massively scalable soft real-time systems with requirements on high availability. Some of its uses are in telecoms, banking, e-commerce, computer telephony and instant messaging. Erlang's runtime system has built-in support for concurrency, distribution and fault tolerance.
+
+> [wikipedia.org/wiki/Erlang_(programming_language)](https://en.wikipedia.org/wiki/Erlang_%28programming_language%29)
+
+%%LOGO%%
+
+# How to use this image
+
+## Create a `Dockerfile` in your Erlang app project
+
+```dockerfile
+FROM erlang:18-onbuild
```

Comment:
You don't have an onbuild tag; probably just add the commands that would be in the onbuild. Something like this?

(warning, I am not an erlang user)
```dockerfile
FROM erlang:18
WORKDIR /usr/src/myapp/
COPY ./ /usr/src/myapp/
CMD ["escript", "myapp.erl"]
```